config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   root 'landing#index'
   resources :books, only: [:index, :show]
+  get 'users/:user_id/rents', to: 'users#rents'
```

Comment:
Use member/collection routes for stuff like this (member in this case). The code in the controller will stay absolutely the same

See `2.10 Adding More RESTful Actions` [here](http://guides.rubyonrails.org/routing.html)